src/Sulu/Bundle/PageBundle/Resources/config/forms/page_excerpt.xml

```diff
@@ -36,6 +36,21 @@
                 <title>sulu_tag.tags</title>
             </meta>
         </property>
+        <property
+            name="ext/excerpt/segment" type="single_select"
+            visibleCondition="__webspace.segments.length &gt; 0"
+        >
+            <meta>
+                <title>sulu_admin.segments</title>
+            </meta>
+            <params>
+                <param
+                    name="values"
+                    type="expression"
+                    value="service('sulu_page.segment_select_helper').getValues(webspace)"
```

Comment:
This line is the problem of [@Quehnie's comment in the ArticleBundle](https://github.com/sulu/sulu/pull/5277#issuecomment-648222022).

I was not aware that this exact form is used in the ArticleBundle. The problem here now is, that they actually need this feature also in the ArticleBundle, but articles don't have a clearly assigned webspace... How should we handle this?

Three possibilities come to mind:

1. Ditch the segments from the webspace and make that a global setting. Would probably help with the project they are currently working on, but might be problematic in other projects.
2. Use the default webspace from the article. This would still be problematic when the article should be shown in multiple webspaces, where each webspace contains segments.
3. Show a separate segment dropdown for each webspace in the excerpt tab of the articles.

Reply:
Have discussed that with @chirimoya, and we will implement option 3 by adding a new field type, which automatically recognizes if a single webspace has been assigned or if the segments for all webspaces should be shown.